salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -8,7 +8,7 @@
 # Various changes to the original are made, based on how we deploy Calico (and
 # its CNI plugins etc.) within MetalK8s.
 
-# It comes from: https://github.com/projectcalico/calico/blob/v3.28.0/manifests/calico.yaml
+# It comes from: https://github.com/projectcalico/calico/blob/v3.28.2/manifests/calico.yaml
 
 ---
 # Source: calico/templates/calico-kube-controllers.yaml
```

Comment:
add this section to BUMPING:
pull from this link 
apply all patches by hand ( diff + good luck have fun)